### PR TITLE
Applied: FHIR-45080 (measurereport-category extension)

### PR DESCRIPTION
**Pull request by yanheras**
https://jira.hl7.org/browse/FHIR-45080

Add an extension "measurereport-category" to the MeasureReport resource.

### CODE REVIEW

File: input/definitions/MeasureReport/CodeSystem-measurereport-category.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  <meta>
+    <profile value="http://hl7.org/fhir/StructureDefinition/shareablecodesystem"/>
+  </meta>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-wg">
+    <valueCode value="cqi"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status">
+    <valueCode value="trial-use"/>
+  </extension>
+  <url value="http://hl7.org/fhir/CodeSystem/measurereport-category"/>
```
Comment (p9g) [inline input/definitions/MeasureReport/CodeSystem-measurereport-category.xml:14]: We were hoping to use http://terminology.hl7.org/CodeSystem/measurereport-category

File: input/definitions/MeasureReport/ValueSet-measurereport-category.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <meta>
+    <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset"/>
+  </meta>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-wg">
+    <valueCode value="cqi"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status">
+    <valueCode value="trial-use"/>
+  </extension>
+  <url value="http://hl7.org/fhir/ValueSet/measurereport-category"/>
```
Comment (p9g) [inline input/definitions/MeasureReport/ValueSet-measurereport-category.xml:14]: we were hoping to use http://terminology.hl7.org/ValueSet/measurereport-category

Comment (p9g) [reply input/definitions/MeasureReport/ValueSet-measurereport-category.xml:14]: I think Yan was having problems building the IG when using the THO URLs for the code system and value set.

**Comment by grahamegrieve** [comment]
Are these problems resolved now?

**Comment by yanheras** [comment]
> Are these problems resolved now?

The pull request is ok. You can merge as is. Thanks.

**Comment by p9g** [comment]
Yan has been conversing with TSMG on this, so please merge as is.

**Comment by yanheras** [comment]
We have communicated with TSMG, they have no concerns of approving this code system staying in the Extension Pack. Given that TSMG won't meet until April 11th, they are good with merging this PR to meet the deadline in order to have this extension be included in this Extension Pack release.

**Review by lmckenzi — APPROVED** [review]

**Pull request merged**